models/brats_mri_generative_diffusion/configs/metadata.json

```diff
@@ -13,7 +14,7 @@
         "1.0.1": "update dependency, update trained model weights",
         "1.0.0": "Initial release"
     },
-    "monai_version": "1.3.1",
+    "monai_version": "1.3.3rc1",
```

Comment:
@KumoLiu I think we should use `1.4rc` here?

Thanks.

Reply:
should use `1.4` here.